changelogs/fragments/fallback_uid.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+    - for those running uids for invalid users (containers), fallback to uid=<uid> when logging fixes #68007
```

Comment:
I think you need to quote this line to include the number as the `#` is turning it into a quote.